src/main/java/net/snowflake/client/jdbc/SnowflakeSQLException.java

```diff
@@ -51,31 +51,43 @@ public SnowflakeSQLException(String queryId, String reason, String sqlState, int
         queryId);
   }
 
-  public SnowflakeSQLException(String reason, String SQLState) {
-    super(reason, SQLState);
+  public SnowflakeSQLException(String reason, String sqlState) {
+    super(reason, sqlState);
     // log user error from GS at fine level
-    logger.debug("Snowflake exception: {}, sqlState:{}", reason, SQLState);
+    logger.debug("Snowflake exception: {}, sqlState:{}", reason, sqlState);
   }
 
+  /** use {@link SnowflakeSQLException#SnowflakeSQLException(String, String, int)} */
+  @Deprecated
```

Comment:
Do we still use it somewhere in our code, or it is only deprecated instead of removed because it is public?

Reply:
I'm not removing it since it's public and it is used in different places